### ISSUE TITLE
hammr account creates still requires certificates for AWS

### ISSUE THREAD

**Issue by alefebvr** [usharesoft/hammr#430]
UForge does not handle certificate files for AWS accounts any more.
However, trying "hammr account create" with an account file missing the certificate fields fails:
```
hammr account create --file awsaccount.yml 
INFO: Checking default credentials file...
INFO: no username provided on command line, trying credentials file
INFO: Using credentials file: /home/alexandre/.hammr/credentials.json
you provided a json file, checking the syntax...
you provided a yaml file, checking the syntax...
ERROR: x509Cert in ami account not found
```

**Comment by alefebvr** [comment]
This is linked to issue #409.
Also, when adding a dummy x509Cert: "dummy.pem" entry (file must exist), further publication fails using this account with message
`Could not read certificate from secret manager. Please contact your UForge administrator.`

**Comment by AliceAlbano** [comment]
Fixed by PR #432
